packages/tenant-process/src/services/tenantService.ts

```diff
@@ -781,6 +769,18 @@ export function tenantServiceBuilder(
         throw attributeRevocationNotAllowed(tenantId, attributeId);
       }
 
+      const isInRevokedBy = verifiedTenantAttribute.revokedBy.some(
+        (a) => a.id === authData.organizationId
+      );
+
+      if (isInRevokedBy) {
+        throw attributeAlreadyRevoked(
+          tenantId,
+          authData.organizationId,
+          attributeId
+        );
+      }
+
```

Comment:
This is the Scala code of this part:

```Scala
      verifier      <- attribute.verifiedBy
        .find(_.id == requesterTenantUuid)
        .toFuture(AttributeRevocationNotAllowed(targetTenantUuid, attributeUuid))
      _             <- Future
        .failed(AttributeAlreadyRevoked(targetTenantUuid, requesterTenantUuid, attributeUuid))
        .unlessA(attribute.verifiedBy.exists(_.id == requesterTenantUuid))
        .whenA(attribute.revokedBy.exists(_.id == requesterTenantUuid))

```

Reply:
Right now, even with this PR, the bug is still present.
Since we are supposedly not removing the entry from the `revokedBy` array when re-verifying, when we verify, revoke and re-verify an attribute, we are in the scenario where, for the same verifier, there is an entry in both `verifiedBy` and `revokedBy` array.  This state leads to the impossibility of revoking again due to the Scala/Node logic we see in this PR

Reply:
This comment is solved by [this](https://github.com/pagopa/interop-be-monorepo/pull/962#discussion_r1754956426) discussion, correct?

Reply:
Yes, the PR is ready to merge